src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -41,12 +41,6 @@ import { Schemas } from 'vs/base/common/network';
 import { VirtualWorkspaceContext } from 'vs/workbench/browser/contextkeys';
 import { formatMessageForTerminal } from 'vs/workbench/contrib/terminal/common/terminalStrings';
 
-
-interface IExtHostReadyEntry {
-	promise: Promise<void>;
-	resolve: () => void;
-}
```

Comment:
Generally keeping interfaces within the single file is better unless they're shared across multiple